content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
 
+![Application and system containers](/static/img/lxd/application-vs-system-containers.svg "Application and system containers")
 
+### Virtual machines vs. system containers
+Virtual machines emulate a physical machine, using the hardware of the host system from a full and completely isolated operating system. System containers, on the other hand, use the OS kernel of the host system instead of creating their own environment. If you run several system containers, they all share the same kernel, which makes them faster and more light-weight than virtual machines.
 
-* Enables **qcow2 backed containers** and regular KVM-like images, giving users more flexibility when deploying their workloads
-* Support for snap installed LXD nodes.
-* Ceph storage pools shared across all virtualization nodes (both KVM and LXD)
-* Seamless LXD public image servers integration in a marketplace-like style. Supports [LXD public image server](https://images.linuxcontainers.org),  [TurnkeyLinux](https://www.turnkeylinux.org), [DockerHub](https://hub.docker.com/) and also the [OpenNebula marketplace](https://marketplace.opennebula.systems/appliance)
-* [Multi-container services deployment](https://docs.opennebula.org/5.8/advanced_components/application_flow_and_auto-scaling/overview.html) with auto-scaling.
-* Heterogeneous hypervisors cloud deployments with KVM, LXD and VMWare vCenter managed by the same cloud orchestration system
+With LXD, you can create both system containers and virtual machines. You should use a system container to leverage the smaller size and increased performance if all functionality you require is compatible with the kernel of your host operating system. If you need a fully isolated environment or functionality that is not supported by the OS kernel of your host system, use a virtual machine.
+
+![Virtual machines and system containers](/static/img/lxd/virtual-machines-vs-system-containers.svg "Virtual machines and system containers")
+
+# Features
+Some of the biggest features of LXD are:
+
+Core API
+: * [Secure by design](https://linuxcontainers.org/lxd/docs/master/security) (through unprivileged containers, resource restrictions, authentication, ...)
+  * [Intuitive](https://linuxcontainers.org/lxd/docs/master/rest-api) (with a simple, clear API and crisp command line experience)
+  * [Scalable](https://linuxcontainers.org/lxd/docs/master/clustering) (from containers on your laptop to clusters of thousands of compute nodes)
+  * [Event based](https://linuxcontainers.org/lxd/docs/master/events) (providing logging, operation, and lifecycle events)
+  * [Remote usage](https://linuxcontainers.org/lxd/docs/master/remotes) (same API used for local and network access)
```

Comment:
I would rather link to the advanced guide: https://linuxcontainers.org/lxd/advanced-guide/#remote-servers

Reply:
For consistency, I wouldn't. As I commented above, we will be working on those docs soon enough and in general they do get updated more than the website. But as I said above, we may be able to de-emphisize the links a bit by not having the whole line link.